tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -695,6 +696,59 @@ class WskBasicUsageTests
             }
     }
 
+    it should "get an action URL" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val actionName = "action name"
+            val packageName = "package name"
+            val defaultPackageName = "default"
+            val webActionName = "web action name"
+            val nonExistentActionName = "non-existence action"
```

Comment:
Also try an action name that has a bunch of special characters to validate the url encoding is correct